src/app/home/home.component.html

```diff
@@ -1,19 +1,17 @@
-<div class="websiteheader">
-  <h2>happy-dev</h2>
-  <p>Creating productive (and happy) developers</p>
-</div>
+<head>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Montserrat:wght@500&family=Nunito+Sans:wght@200&display=swap" rel="stylesheet">
+</head>
 
-<div class="header">
-  <h1>
-    We're so happy that you're here!
-  </h1>
-    happy-dev is a productivity application with a twist! Track your to-do lists, time spent on work tasks (or personal tasks) and how effective you are in completing daily tasks, while also measuring a developer's happiness levels. Being a software developer can be hard...you can use happy-dev to help with work tasks, while also measuring your happiness levels, detecting your interest in projects, and analyzing your satisfaction with work, overall.
-</div>
+<h1>happy-dev</h1>
 
-<div class="footer">
-  <p>Thanks for visisting happy-dev!</p>
-</div>
+<h2>Creating happy (and productive) developers</h2>
 
-<main>
-  <router-outlet></router-outlet>
-</main>
+<div class="homePage">
+  <h3>happy-dev is a productivity application with a twist! Track your to-do lists,
+    time spent on work tasks (or personal tasks) and how effective you are in completing daily tasks,
+    while also measuring a developer's happiness levels. Being a software developer can be hard...
+    you can use happy-dev to help with work tasks, while also measuring your happiness levels, detecting your
+    interest in projects, and analyzing your satisfaction with work, overall.</h3>
```

Comment:
Try not to use heading elements purely for styling. They are intended for hierarchy. for example you may do something like this:

``` html
<h1>My Title</h1>
<section>
    <h2>Section Title</h2>
    <p>Some content here</p>
    
    <h3>Deeper dive into content</h3>
    <ul>
        <li></li>
        <li></li>
    </ul>
</section>
````

If you want to style it differently instead use a class and some CSS.

Reply:
I won't hold this up just for styling. You can make some of these changes in the next PR.